miner/payload_building.go

```diff
@@ -80,8 +98,9 @@ func (payload *Payload) update(block *types.Block, fees *big.Int, elapsed time.D
 		payload.fullFees = fees
 
 		feesInEther := new(big.Float).Quo(new(big.Float).SetInt(fees), big.NewFloat(params.Ether))
-		log.Info("Updated payload", "number", block.NumberU64(), "hash", block.Hash(),
-			"txs", len(block.Transactions()), "gas", block.GasUsed(), "fees", feesInEther, "elapsed", common.PrettyDuration(elapsed))
+		log.Info("Updated payload", "id", payload.id, "number", block.NumberU64(),
```

Comment:
I would still prefer to keep block-number and block-hash together. Note the block-hash is actually the final execution-block-hash here.